### PR TITLE
chore: set max SDK version to <3.0.0

### PR DESCRIPTION
**Pull request by chalin**
cc @kwalrath @kevmoo

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.14.5
```
Comment (kevmoo) [inline CHANGELOG.md]: Do 0.14.4+1

**Review by kevmoo — APPROVED** [review]
One nit...

File: CHANGELOG.md (hunk 1 above)
Comment (chalin) [reply CHANGELOG.md]: Done

File: analysis_options.yaml [hunk 2]
```diff
@@ -1,2 +1 @@
 analyzer:
```
Comment (kevmoo) [inline analysis_options.yaml]: Just delete the file!

**Review by kevmoo — APPROVED** [review]

File: analysis_options.yaml (hunk 2 above)
Comment (chalin) [reply analysis_options.yaml]: Ok, if that is what you prefer. (My reasoning is that it isn't harmful to keep, and it is easier to add options later on than to try to remember the file name, etc.)

Comment (chalin) [reply analysis_options.yaml]: Done

Comment (kevmoo) [reply analysis_options.yaml]: Fair – but having an empty file is just confusing for someone reading the code. That's my default...fewer questions for code skimmers

Comment (chalin) [reply analysis_options.yaml]: Got it

**Pull request merged**